_docs/templates/email-template.md

```diff
@@ -379,8 +486,17 @@ For now, though, here are a few of the completed studies:
 - [Summary of research done on the Notification Project](http://design.rax.io/?p=2380)
 - [Improving how Rackspace Communicates with its customers](http://design.rax.io/?p=2387)
 
+{% endcolumn %}
+</div>
+
 ## Visual examples
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
 
 {% figure [caption:"Example of a billing email using these guidelines"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/layout/email-templates/Email%20Detail.svg){:width="60%"}
+![]({{site.cdn_url}}/img/layout/email-templates/Email%20Detail.svg)
```

Comment:
<img width="986" alt="screen shot 2017-11-20 at 11 18 32 am" src="https://user-images.githubusercontent.com/1538692/33031699-9e77ebc6-cde4-11e7-942e-3194c4a0b7d8.png">

Aside from the spacing, check spelling (`prodcut` again, `tickts`)